### ISSUE TITLE
import pandas as pd

### ISSUE THREAD

**Issue by Wh1t3z2** [Wh1t3z2/Rookie-#1]
from sklearn import linear_model
data = pd.read_csv("/home/aistudio/data/data4698/simple_example.csv")
#print(data["x"],data["y"],data)
features = data['x'] 
label = data['y']
train_dataset = data[:15] #训练集
test_dataset = data[15:] #测试集
#Creat linear regression model
moel = linear_model.LinearRegression()
model.fit(train_dataset[features],train_dataset[label])
